index.html

```diff
@@ -37,6 +41,7 @@ <h4 class="connect">LET'S CONNECT</h4>
 </div>
   </header>
    <Section class="containers">
+    
     <div class="container">
         <div class="image">
             <img class="img" src="./images/tonic.png" alt="">
```

Comment:
- [ ] 🙏Please adjust the padding and margin of your design🎁 to match the Figma design. This will enhance the appearance of your👩‍🦰 portfolio and make it look as cool as the Figma design.
 
 Your design:

![image](https://user-images.githubusercontent.com/63445083/236599274-024e2f04-3a8c-4f2d-b2f7-aaadd3cf036e.png)

Figma design:

![image](https://user-images.githubusercontent.com/63445083/236599517-efd5735a-a125-4436-bc40-206235f98278.png)

Reply:
As requested by the previous Code Reviewer please consider making the margin of the container from all its sides similar to the design to make it look alike to the design.

### IMAGE DEMONSTRATING MARGINS:-

![image](https://user-images.githubusercontent.com/94223367/236618455-ad392b9e-5530-4cd4-87a3-eef7b1d088c6.png)

```
margin: 142px;
```